libasn1print/asn1printproto.c

```diff
@@ -318,9 +437,22 @@ asn1print_expr_proto(asn1p_t *asn, asn1p_module_t *mod, asn1p_expr_t *expr,
 		}
 
 		proto_msg_def_t *msgelem = proto_create_msg_elem("value", "int32", NULL);
+
 		switch (expr->expr_type) {
 		case ASN_BASIC_INTEGER:
 			if (expr->constraints != NULL) {
+                // adding APER tags
```

Comment:
your new lines use whitespace and existing code uses tabs - I thought CLion would replace the whitespace with tabs automatically, but it looks like you will have to

Reply:
done